great_expectations/rule_based_profiler/parameter_builder/value_set_multi_batch_parameter_builder.py

```diff
@@ -136,8 +146,8 @@ def _build_parameters(
         )
         details: dict = parameter_node[FULLY_QUALIFIED_PARAMETER_NAME_METADATA_KEY]
 
-        unique_values: Set[Any] = _get_unique_values_from_nested_collection_of_sets(
-            collection=metric_values
+        unique_values: Set = _get_unique_values_from_nested_collection_of_sets(
```

Comment:
@Kilo59 I believe that in this case, we should just use `set` (instead of `typing.Set).  Thank you.

Reply:
There is no `V` type var at this scope.
`Set`, `set`, and `Set[Any]` are all equivalent.

Reply:
@Kilo59 Fine -- to me, having `Set` without anything in the brackets seems sloppy, but fine.  Perhaps if you make another change to this PR, you could change it to `set`; otherwise, we can leave it be (or do it later).  Thanks.

Reply:
changed to `set`